dev_playbook.yml

```diff
@@ -96,3 +96,27 @@
       become: yes
       become_user: root
 
+    # Nginx logrotate: keep nginx logs forever on /mnt disk
+    - name: Make sure extra nginx log dirs exist
+      file:
+        state: directory
+        path: "{{ item }}"
+      with_items:
+        - nginx_log_olddir_root
+        - nginx_long_term_log_dir
+    - name: Add lines to main block in nginx logrotate conf
+      lineinfile:
+        path: /etc/logrotate.d/nginx
+        line: "{{ item }}"
+        insertafter: ".*{"
+      with_items:
+        - "\tdateext"
```

Comment:
Not sure what dataext does but I assume that's necessary.

Reply:
```
dateext
Archive old versions of log files adding a daily extension like YYYYMMDD instead of simply adding a number. The extension may be configured using the dateformat option.
```

At the moment our log files are numbered so without adding the date we'd have no way to ID them long term, plus they'd overwrite each other